### PR TITLE
feat(config): add support for Sensative AB Strips Drip 700

### PR DESCRIPTION
**Pull request by popescunsergiu**

### CODE REVIEW

**Comment by popescunsergiu** [comment]
Add support for Sensative AB Strips Drip 700
Update Sensative AB - Strips Guard 700 filename and label according to manufacturer specifications

**Comment by popescunsergiu** [comment]
@AlCalzone may I please have some help with this. I simply don't understand what is wrong with my commit message

**Comment by popescunsergiu** [comment]
Add support for Sensative AB Strips Drip 700
Update Sensative AB Strips Guard 700 label based on manufacturer documentation

**Comment by AlCalzone** [comment]
`Add` must be lowercase. If you edit the PR title and add another commit, the check should be ok.

File: packages/config/config/devices/0x019a/11_04_022.json [hunk 1]
```diff
@@ -0,0 +1,233 @@
+{
+	"manufacturer": "Sensative AB",
+	"manufacturerId": "0x019a",
+	"label": "11-01-021",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019a/11_04_022.json]: Something is off here. Filename `11_04_022`, but label `11-01-021`.

File: packages/config/config/devices/0x019a/11_04_022.json [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019a/11_04_022.json]:
```suggestion
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
```
please carry througout

File: packages/config/config/devices/0x019a/11_04_022.json [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+			"label": "Leakage/Moisture Reporting Period",
+			"valueSize": 1,
+			"unit": "hours",
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 0
+		},
+		"15": {
+			"label": "Activate Supervision",
+			"description": "Activate Supervision command for only important alarm events or all events. *Supervised commands require a confirmation from the gateway when a notification is received.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019a/11_04_022.json]:
```suggestion
			"description": "Request confirmation from the gateway that a command was received.",
```

File: packages/config/config/devices/0x019a/11_04_022.json [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+				},
+				{
+					"label": "Alarm Report (*)",
+					"value": 1
+				},
+				{
+					"label": "All Reports",
+					"value": 2
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019a/11_04_022.json]:
```suggestion
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Alarm Report",
					"value": 1
				},
				{
					"label": "All Reports",
					"value": 2
				}
			]
```

File: packages/config/config/devices/0x019a/11_04_022.json [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+				{
+					"label": "LED Off",
+					"value": 0
+				},
+				{
+					"label": "LED On",
+					"value": 1
+				}
+			]
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019a/11_04_022.json]: We're not a fibaro gateway, so we might even leave this parameter out.

File: packages/config/config/devices/0x019a/11_04_022.json [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+		"23": {
+			"label": "Perform Leakage/Moisture Sensor Calibration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Device resets to this value after performing capacitate calibration and sends a report with value 0 to the gateway",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019a/11_04_022.json]:
```suggestion
					"label": "Idle",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x019a/11_04_022.json (hunk 5 above)
Comment (popescunsergiu) [reply packages/config/config/devices/0x019a/11_04_022.json]: Done. Thx

**Comment by darkbasic** [comment]
Are Sensative Strips Guard 700 already supported or do they need a PR as well? Because I've just bought one and I'd like to test it.

**Review by AlCalzone — APPROVED** [review]
OK with my changes

**Pull request merged**